filebeat/fileset/factory.go

```diff
@@ -18,6 +18,7 @@ type Factory struct {
 	registrar             *registrar.Registrar
 	beatVersion           string
 	pipelineLoaderFactory PipelineLoaderFactory
+	updatePipelines       bool
```

Comment:
Should you directly specify the update-pipeline flag here instead of passing through all instances? It would simplify the code I think.

Reply:
I would rather keep it this way. If I moved the flag here, the code would get messier places. This way, global flags could be in the same place and can be tracked easily to the place they are used.

Reply:
Got it, I missed the part that you also introduced it in the commands list.